live-site/components/judging-section/judging-schedule-welcome/JudgingScheduleWelcome.tsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import { StyledBackButton, StyledCenteredDropdown, StyledJudgingScheduleSection, StyledWelcomePerson } from '../judging-schedule-welcome/JudgingScheduleWelcome.styles';
+import { StyledSecondaryDropdown } from '../../../../shared-ui/components/secondary-dropdown/SecondaryDropdown.styles';
+import PrimaryButton from '../../../../shared-ui/components/primary-button/PrimaryButton';
+import { JudgingScheduleWelcomeProps } from '../../../lib/types';
+
+const JudgingScheduleWelcome: React.FC<JudgingScheduleWelcomeProps> = ({ schedulePersonType }) => {
+  return (
+    <div>
+      <StyledBackButton> 
+        <PrimaryButton btnText='back' btnLink='../' /> 
```

Comment:
change to ./ if you are meaning to go back to the selection page

Reply:
./ means the current directory ../ means the parent of the current directory, so this actually works right due to the extra folder

Reply:
ohhh rightt lol

Reply:
I was taking be all the way back home when I tested it on my machine

Reply:
omg cool hf in pittsburgh :D